presqt/api_v1/urls.py

```diff
@@ -13,5 +13,7 @@
 
     # Resources
     path('targets/<str:target_name>/resources/',
-         ResourceCollection.as_view(), name="resource_collection")
+         ResourceCollection.as_view(), name="resource_collection"),
+    path('targets/<str:target_name>/resources/<str:resource_id>/',
+         Resource.as_view(), name="resource")
```

Comment:
I know it's not in this file, but can we route the presqt-{whatever}.crc.nd.edu to the main api page instead of the django error that currently pops up?